web/frontend/src/components/slam/index.svelte

```diff
@@ -468,12 +477,14 @@ useConnect(() => {
       <div class="flex flex-col gap-6 pb-4">
         {#if overrides?.isCloudSlam && overrides.mappingDetails}
           <header class="flex flex-col justify-between gap-3 text-xs">
-            <div class="flex flex-col">
-              <span class="font-bold text-gray-800">Mapping mode</span>
-              <span class="capitalize text-subtle-2"
-                >{overrides.mappingDetails.mode}</span
-              >
-            </div>
+            {#if overrides.mappingDetails.mode}
```

Comment:
I am not displaying the "Mapping mode" on the RC card for non-Cartographer modules, since they don't have a `mode`.

Reply:
can we use the value returned from `Properties`

Reply:
I think so. Is it ok to open a separate ticket for this? I think the logic here can stay, in case `mode` is still undefined.

Reply:
making another ticket to pull the mapping mode out of properties is fine

Reply:
Created ticket: https://viam.atlassian.net/browse/RSDK-7061